src/core/controller/index.ts

```diff
@@ -1910,9 +1910,46 @@ Here is the project's README to help you get started:\n\n${mcpDetails.readmeCont
 		const localClineRulesToggles =
 			((await getWorkspaceState(this.context, "localClineRulesToggles")) as ClineRulesToggles) || {}
 
+		// Get live model info from the active task's API handler, if available
```

Comment:
I don't understand this explain

Reply:
I thought I needed all provider configs, but it seems this was a mistake. I've removed everything not pertaining to litellm.